### DISCUSSION TITLE
检测模型的main_indicator参数可以设置为什么？

### DISCUSSION THREAD

**Discussion by wq881103** [PaddlePaddle/PaddleOCR#13166] [Q&A]
默认参数是hmean，还支持什么参数

**Comment by SWHL** [comment]
建议去源码里翻翻，目前PaddleOCR项目文档还有些残废

**Comment by UserWangZz** [comment]
ppocr/metrics/det_metric.py
main_indicator可以设置为precision，recall，hmean，不过看代码，该字段应该只在训练过程中作为保存best_accuracy模型的指标起作用。
